pandas/tests/frame/indexing/test_xs.py

```diff
@@ -327,10 +337,12 @@ def test_xs_droplevel_false(self):
         expected = DataFrame({"a": [1]})
         tm.assert_frame_equal(result, expected)
 
+    # TODO(ArrayManager) xs with axis=1 can return view
+    @td.skip_array_manager_not_yet_implemented
     def test_xs_droplevel_false_view(self):
         # GH#37832
         df = DataFrame([[1, 2, 3]], columns=Index(["a", "b", "c"]))
         result = df.xs("a", axis=1, drop_level=False)
-        df.values[0, 0] = 2
```

Comment:
this version feels much clearer as to whats going on

Reply:
Do you have a suggestion for how to then do it? (we can't rely on .values to mutate)

Reply:
i guess what we really care about here is `np.shares_memory(result._values, df.iloc[:, 0])`

Reply:
Added the `np.shares_memory` check. And turned out that also with ArrayManager this actually gives a view. But it's the `iloc` modifying the parent that doesn't result in modifying the child result, when using ArrayManager, but also when using a mixed DataFrame. So added that as an explicit test case as well.